### DISCUSSION TITLE
After switching on multi-user I get Authentiaction Exception?

### DISCUSSION THREAD

**Discussion by Terry-JF** [firefly-iii/firefly-iii#9101] [Firefly III] [answered]
I enabled multi-user yesterday and created a second user account. This morning cron ran the normal task and imported my recent transactions via GoCardless as usual, I have the emails to confirm this. However the script on my Windows PC that uses Curl to subsequently fetch these transaction from FF3 via the API failed with the 'Unauthenticated' message returned. The PC script is automated by Task Scheduler and has been working fine for several days now using a Personal Access Token.

Has setting up multi-user caused this issue, and how to fix it?

Thanks, Terry

**Comment by Terry-JF** [comment]
I don't think this had anything to do with turning on multi-user, as turning it off again had no effect. Then I read this post #4595 where it mentions that Personal Access Tokens expire, so I created a new PAT and now everything works again.

Does anybody know how long a PAT is valid for, is it 30 days? I feel like it was less than that since I created the expired one.

**Reply by JC5** [reply]
Sorry, they should be valid even longer, I'm not sure what the root cause could be.

**Comment by Terry-JF** [comment, marked as answer]
I see in #4595 it shows a returned line like - "expires_in": 1209600,

This is the number of seconds in 14 days so I guess that the Personal Access Tokens expire after 14 days.

**Reply by JC5** [reply]
That only applies to tokens that are generated from a refresh token. I'm not sure how long the other PATs are valid, but longer than 30 days for sure.
